projects/vdk-heartbeat/src/vdk/internal/heartbeat/vdk_heartbeat_data_job/trino_ingestion/10_verify_data.py

```diff
@@ -0,0 +1,45 @@
+# Copyright 2021 VMware, Inc.
+# SPDX-License-Identifier: Apache-2.0
+import logging
```

Comment:
I think it will be better to be 20_verify_data . I know that V is after I so the order is preserved but it's very non-obvious. I had to think about it.

Reply:
Fixed